app/mailers/transaction_mailer.rb

```diff
@@ -30,7 +30,7 @@ def transaction_preauthorized(transaction)
       payment_type = MarketplaceService::Community::Query.payment_type(@community.id)
       gateway_expires = MarketplaceService::Transaction::Entity.authorization_expiration_period(payment_type)
 
-      expires = Maybe(transaction).booking.end_on.map { |booking_end|
+      expires = Maybe(transaction).booking.end_on_exclusive.map { |booking_end|
```

Comment:
I'm I correct that this code has actually been broken for day pickers until now? There's no check of unit type and the assumption seems to have already been end_on is exclusive.